disperser/batcher/txn_manager.go

```diff
@@ -83,6 +87,7 @@ func NewTxnRequest(tx *types.Transaction, tag string, value *big.Int, metadata i
 		Metadata: metadata,
 
 		requestedAt: time.Now(),
+		txAttempts:  make([]*types.Transaction, 0),
```

Comment:
The passed in tx should be in this list? It's an attempted one as well

Reply:
Yes, it should be in the list as well